site/themes/blogtxt/templates/base.tmpl

```diff
@@ -6,8 +6,12 @@ ${base.html_headstart()}
 ### Leave this block alone.
 </%block>
 ${template_hooks['extra_head']()}
+
 <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
-<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
+<link href="${rel_link(permalink,'/assets/css/tagcloud-es-small.css')}" rel="stylesheet" type="text/css">
+
+<!-- <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script> -->
```

Comment:
Por que jquery-ui está ahora comentado?

Reply:
Because it was just needed for accordion widget. 
I will erase it.